src/DataLayouts/DataLayouts.jl

```diff
@@ -2330,8 +2330,17 @@ function ColumnMask(
     @assert Nq isa Integer
     @assert Nh isa Integer
     T = horizontal_layout_type
-    is_active = replace_basetype(T{FT, Nq}(Array{FT}, Nh), Bool)
-    Nijh = Nq * Nq * Nh
+    is_active = replace_basetype(T{FT, Nq}(DA{FT}, Nh), Bool)
+    @. is_active = true
```

Comment:
Is this the part that sets the mask = 1 everywhere if enable_mask = true?

Reply:
Yes